scripts/run_evolution.sh

```
@@ -24,20 +24,20 @@ if [ ! -d ".venv" ]; then
-if [ ! -f ".venv/bin/mad_spark_alt" ]; then
-    echo "Error: mad_spark_alt not found in virtual environment"
+# Check if msa is installed
+if [ ! -f ".venv/bin/msa" ]; then
+    echo "Error: msa not found in virtual environment"
     echo "Please run 'uv pip install -e .' first"
     exit 1
 fi
 
 echo "Starting Mad Spark Alt with evolution..."
-echo "Command: .venv/bin/mad_spark_alt $@"
+echo "Command: .venv/bin/msa $@"
```

Shellcheck (0.11.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)